instrumentation-core/spring/README.md

```diff
@@ -153,7 +153,7 @@ Here we will create rest controllers for `FirstService` and `SecondService`.
 
 Required dependencies and configurations for FirstService and SecondService projects can be found [here](#setup-for-manual-instrumentation).
 
-### FirstService
+### Instrumentation of Receiving Service: FirstService
```

Comment:
i think the names "Receiving Service" and "Client Service" are not too clear

i like the idea of trying to give them more descriptive names, instead of `FirstService` and `SecondService` though

maybe `MainService` / `TimeService`?

or `FrontService` / `TimeService` ?

Reply:
MainService and TimeService sounds good